apps/editing-toolkit/editing-toolkit-plugin/wpcom-block-editor-nux/class-wp-rest-wpcom-block-editor-nux-status-controller.php

```diff
@@ -11,6 +11,8 @@
  * Class WP_REST_WPCOM_Block_Editor_NUX_Status_Controller.
  */
 class WP_REST_WPCOM_Block_Editor_NUX_Status_Controller extends \WP_REST_Controller {
+	const NEW_SITE_AGE_SECONDS = 30 * 60;
```

Comment:
Maybe a comment here about the reasoning for choosing 30 minutes, e.g. for the /start flow where people might need time before they enter the editor,   and maybe link to P2 discussion (if that is normal for including in comments?)

Reply:
I added a comment justification, yes I see we've linked to p2s in other places as well 👍 

I also improved some other comments in the code